corehq/apps/analytics/static/analytix/js/hubspot.get_demo.js

```diff
@@ -0,0 +1,41 @@
+/* global window, hbspt */
+$(function () {
+    'use strict';
+
+    hqImport("analytix/js/hubspot").then(function () {
+        var hubspotGet = hqImport('analytix/js/initial').getFn('hubspot'),
+            hubspotApiId = hubspotGet('apiId');
+
+        // Old (control) hubspot form:
+        hbspt.forms.create({
+            portalId: hubspotApiId,
+            formId: "0f5de42e-b562-4ece-85e5-cfd2db97eba8",
+            target: "#get-demo-cta-form-body",
+            css: "",
+        });
+
+        // New (variant A) hubspot form:
+        hbspt.forms.create({
+            portalId: hubspotApiId,
+            formId: "38980202-f1bd-412e-b490-f390f40e9ee1",
+            target: "#get-demo-cta-form-content",
+            css: "",
+            onFormSubmit: function ($form) {
+                $('#get-demo-cta-calendar-content').fadeIn();
+                $('#get-demo-cta-form-content').addClass('hide');
+                var email = $form.find('[name="email"]').val(),
+                    firstname = $form.find('[name="firstname"]').val(),
+                    lastname = $form.find('[name="lastname"]').val(),
+                    newUrl = document.location.href + '?email=' + email + '&name=' + firstname + '%20' + lastname;
+
+                window.history.pushState(
+                    "dimagi-contact-url " + document.title, document.title, newUrl
+                );
+
+                $.getScript('//cdn.scheduleonce.com/mergedjs/so.js');
```

Comment:
This causes the schedule once widget to pop up as soon as this script loads, right? And that's just a widget, it doesn't leave the page? It's be good to put a comment here, since it's a bit odd to see a `getScript` call without a callback.